torch_frame/datasets/tabular_benchmark.py

```diff
@@ -386,14 +386,9 @@ class TabularBenchmark(torch_frame.data.Dataset):
     base_url = 'https://huggingface.co/datasets/inria-soda/tabular-benchmark/raw/main/'  # noqa
     # Dedicated URLs for large datasets
     base_url_large = 'https://huggingface.co/datasets/inria-soda/tabular-benchmark/resolve/main/'  # noqa
+    name_list = sorted(list(name_to_task_category.keys()))
 
-    @classmethod
-    @property
```

Comment:
classmethod + property is deprecated in Python 3.11 https://github.com/python/cpython/pull/92379